language/de/acp/board.php

```diff
@@ -348,11 +348,15 @@
 
 // Cookie Settings
 $lang = array_merge($lang, array(
-	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'Hier legst du die Einstellungen fest, die verwendet werden, um Cookies an die Browser deiner Benutzer zu senden. In den meisten Fällen sollten die Standardwerte ausreichend sein. Führe Änderungen mit Bedacht durch, fehlerhafte Einstellungen könnten deine Benutzer daran hindern, sich anzumelden.',
+	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'Hier legst du die Einstellungen fest, die verwendet werden, um Cookies an die Browser deiner Benutzer zu senden. In den meisten Fällen sollten die Standardwerte ausreichend sein. Führe Änderungen mit Bedacht durch, fehlerhafte Einstellungen könnten deine Benutzer daran hindern, sich anzumelden. Wenn du Probleme mit Benutzern hast, die nicht eingeloggt bleiben, besuche die <b><a href="https://www.phpbb.com/support/go/cookie-settings/">phpBB.com Knowledge Base - Fixing incorrect cookie settings</a></b>.',
```

Comment:
Ich würde "nicht angemeldet" statt "nicht eingeloggt" verwenden und wie in den anderen Fällen auch zusätzlich auf einen deutschen Artikel verlinken,